app/models/series.rb

```diff
@@ -104,4 +108,23 @@ def schedule_for_episode_number(episode_number)
 
     schedule_index
   end
+
+  def destroy
+    if v4?
+      really_destroy!
```

Comment:
I guess I thought we were moving away from `paranoid` in v4, to having things actually deleted.  Maybe I mis-remembered that conversation.